doc-site/components/nve-switch.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: component
+---
+
+<CodeExamplePreview>
+
+```html
+<nve-switch></nve-switch>
+```
+
+</CodeExamplePreview>
+
+## Eksempler
+
+Med ikoner:
+<CodeExamplePreview>
+
+```html
+<nve-switch>
+  <nve-icon slot="officon" name="light_mode"></nve-icon>
+  <nve-icon slot="onicon" name="dark_mode"></nve-icon>
+</nve-switch>
+```
+
+</CodeExamplePreview>
+
+Label:
+<CodeExamplePreview>
+
+```html
+<nve-switch>
+  <div slot="label">Slå på</div>
+</nve-switch>
```

Comment:
Har du testet hva som skjer hvis du legger tekst inni sloten uten navn?
`<nve-switch>Ledetekst</nve-switch>`

Trenger du egentlig å navngi label-sporet?
Jeg spør fordi jeg mener jeg har sett flere NVE- og Shoelace-komponenter som bruker "standard-sporet" til tekst-innhold etc, f.eks. nve-button

Reply:
Endrer slot label til å være default

Reply:
Kult! 
Må det være en div rundt label, slik du har [her](https://brave-meadow-0c645bd03-206.westeurope.5.azurestaticapps.net/components/nve-switch.html#label-vises-bak-bryter)? Jeg bare spør:)

Reply:
Nei, trenger ikke det. Fjernet den for å gjøre det klart